main/ServiceManager/ServiceManager.cpp

```diff
@@ -3,6 +3,16 @@
 #include "SharedBus.h"
 #include "GUI.h"
 
+// ****************************** Local Variables
+typedef enum 
+{
+    IDLE = 0,
+    INIT,
+    START,
+    ACTIVE, 
+}ServiceManagerDaemonState;
```

Comment:
Since this is a local enum (not refrenced anywhere else), you can drop the `ServiceManager` at the beginig to shorten it.

Reply:
it was fixed